lua/lvim/core/cmp.lua

```diff
@@ -329,10 +326,15 @@ M.config = function()
             confirm_opts.behavior = cmp.ConfirmBehavior.Insert
           end
           if cmp.confirm(confirm_opts) then
+            if jumpable() then
```

Comment:
```suggestion
            if jumpable(1) then
```

Reply:
1 is optional

Reply:
yeah, I know but it's a bit easier to read, i.e. more consistent, since the very next command is `luasnip.jump(1)`